src/app/orders/order-detail/order-detail.page.ts

```diff
@@ -1,15 +1,148 @@
+import { ProfileModel } from 'src/providers/models/profile.model';
 import { Component, OnInit } from '@angular/core';
+import { OrderService } from '../../../providers/services/orders/order.service';
+import { ActivatedRoute, Router } from '@angular/router';
+import { StorageServiceProvider } from '../../../providers/services/storage/storage.service';
+import { CONSTANTS } from '../../../providers/utils/constants';
+import { WidgetUtilService } from '../../../providers/utils/widget';
 
 @Component({
   selector: 'app-order-detail',
   templateUrl: './order-detail.page.html',
   styleUrls: ['./order-detail.page.scss'],
 })
 export class OrderDetailPage implements OnInit {
+  orderItems: any = [];
+  orderDetail: any = {};
```

Comment:
Didn't we have any model for order detail object?

Reply:
use order model.

Reply:
I used orderItem model.